src/workers/sourceSquadCreated.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  getContactIdByEmail,
+  addUserToContacts,
+  LIST_DRIP_CAMPAIGN,
+} from '../common';
+import { Source, SourceMember, User, UserActionType } from '../entity';
+import { SourceMemberRoles } from '../roles';
+import { insertOrIgnoreAction } from '../schema/actions';
+import { ChangeObject } from '../types';
+import { messageToJson, Worker } from './worker';
+
+interface Data {
+  source: ChangeObject<Source>;
+}
+
+const worker: Worker = {
+  subscription: 'api.source-squad-created',
```

Comment:
Since sendgrid api is not the most stable one I'd suggest to separate this to two workers

Reply:
Sure, kinda same thinking from this comment https://github.com/dailydotdev/daily-api/pull/1422#discussion_r1316902768 do we prefer more different subscriptions/different workers?

Reply:
I usually follow the "do only one thing" approach for workers. We have to think about failures and retries which often happen

Reply:
> do only one thing

👌